packages/devtools_app/lib/src/screens/debugger/codeview.dart

```diff
@@ -1154,7 +1154,7 @@ class _LineItemState extends State<LineItem>
 
     if (word != '') {
       try {
-        final response = await controller.evalAtCurrentFrame(word);
+        final response = await controller.evalService.evalAtCurrentFrame(word);
```

Comment:
This case is tricky as you wouldn't want any injected variables from the console to show up when evaluating while hovering over source code. Maybe this case should take an extra named parameter like `includeConsoleVariables: false`

Reply:
Discussed and agreed `evalAtCurrentFrame` will never be invoked for global scope.